src/content/en/updates/2017/06/headless-karma-mocha-chai.md

```diff
@@ -0,0 +1,137 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Testing on Headless Chrome with Karma/Mocha/Chai"
+
+{# wf_updated_on: 2017-06-13 #}
+{# wf_published_on: 2017-06-13 #}
+
+{# wf_tags: headless,testing,karma,mocha,chai #}
+{# wf_featured_image: /web/updates/images/generic/headless-chrome.png #}
+{# wf_featured_snippet: How to setup Karma, Mocha, and Chai to run automated tests on Headless Chrome. #}
+
+# Testing on Headless Chrome with Karma/Mocha/Chai {: .page-title }
```

Comment:
Actually, I'll just let you do it if you want. Check out the "Do you identify as a developer, designer, or something else?" question at the top of this doc for an example: https://developers.google.com/web/tools/chrome-devtools/css/

If you want to add one to this post, just paste this into the doc

```
{% framebox width="auto" height="auto" enable_widgets="true" %}
<script>
var response = 'Thanks for the feedback!';
var feedback = {
  "category": "Headless Chrome",
  "question": "How much experience do you have with automated testing?",
  "choices": [
    {
      "button": {
        "text": "None"
      },
      "response": response,
      "analytics": {
        "label": "Automated Testing / Experience / None"
      }
    },
    {
      "button": {
        "text": "Some"
      },
      "response": response,
      "analytics": {
        "label": "Automated Testing / Experience / Some"
      }
    },
    {
      "button": {
        "text": "A Lot"
      },
      "response": response,
      "analytics": {
        "label": "Automated Testing / Experience / A Lot"
      }
    }
  ]
};
</script>
{% include "web/_shared/multichoice.html" %}
{% endframebox %}
```